submarine-sdk/pysubmarine/submarine/experiment/api/experiment_client.py

```diff
@@ -110,19 +113,19 @@ def delete_experiment(self, id):
         response = self.experiment_api.delete_experiment(id)
         return response.result
 
-    def get_log(self, id, master=True):
+    def get_log(self, id, master=False):
         """
         Get training logs of the experiment.
         By default only get the logs of Pod that has labels 'job-role: master'.
-        :param master: By default get pod with label 'job-role: master' pod if True.
+        :param master: Get pod with label 'job-role: master' pod if True.
```

Comment:
The description of "master" param seems confusing. I thought get_log(id) will get all pod's log, and get_log(id, master=false) will discard the master pod log.
But per the description, get_log(id, master=True) will get a master log. This is confusing.

> Get pod with label 'job-role: master' pod if True.

Reply:
I just follow the convention in `tf-operator` 
https://github.com/kubeflow/tf-operator/blob/cf6ebd6d2d9f679646dad3530e9314f63505f98a/sdk/python/kubeflow/tfjob/api/tf_job_client.py#L356-L371
Maybe we could change the description, so it will be more clear for the users.
Any suggestion for the comments and param in `get_log`

Reply:
How about this?
```Python
"""
get_logs(self, id, onlyMaster=False)   
Get training logs of all pod of the experiment.
By default get all the logs of Pod
:param id: experiment id
:param onlyMaster: By default include pod log of "master" which might be Tensorflow PS/Chief or PyTroch master
:return: str: pods logs
"""
```

Reply:
Thanks, @tangzhankun.
it's more clear than before. I've updated it.